zadanie 1

```diff
@@ -0,0 +1,21 @@
+def sort_letters(string)
+	string.chars.sort.join
+end
+
+def count_vowels(string)
+	string.count('aeiouy')
+end
+
+def filter_odd(array)
+	array.slelect(&:odd?)
```

Comment:
plik nie ma rozszerzenia .rb i masz literówkę w tym miejscu ;)

Reply:
Witam, 
udało mi się już to poprawić :)
Dzięki za uwagi :)

Reply:
No i super :)